test/tts_pipeline/test_tts_engine.py

```diff
@@ -184,6 +184,8 @@ def _gen_hello_hiho_query() -> AudioQuery:
         volumeScale=1.3,
         prePhonemeLength=0.1,
         postPhonemeLength=0.2,
+        pauseLength=None,
+        pauseLengthScale=1.0,
```

Comment:
テストの中の値、周りに合わせてくださってありがとうございます！

ここはいろんな値が入るぽいところっぽい？から、ちょっと変わった値入れてもいいかも。
（もちろんそのままでも問題ないと思いますが）
```suggestion
        pauseLength=0.3,
        pauseLengthScale=0.8,
```